esx_service/cli/vmdkops_admin.py

```diff
@@ -1034,5 +1106,223 @@ def tenant_access_ls(args):
     rows = generate_tenant_access_ls_rows(privileges)
     print(cli_table.create(header, rows))
 
+# ==== CONFIG DB manipulation functions ====
+
+def create_db_symlink(path, link_path):
+    """Force-creates a symlink to path"""
+    if os.path.islink(link_path):
+        os.remove(link_path)
+    try:
+        os.symlink(path, link_path)
+    except Exception as ex:
+        print("Failed to create symlink at {} to {}".format(auth_data.AUTH_DB_PATH, path))
+        sys.exit(ex)
+
+
+def db_move_to_backup(path):
+    """Saves a DB copy side by side. Basically, glorified copy"""
+    print("Warning: full DB backup is not implemented yet. Copying {} to .bck".format(path))
+    #TODO:
+    # - get a name as path.bck-date
+    # - copy a file, yell if it's problem (just FYI)
+    shutil.move(path, path + ".bck")
+    return
+
+
+def is_local_vmfs(datastore_name):
+    """return True if datastore is local VMFS one"""
+    # TODO - check for datastore being on local VMFS volume:
+    # si = pyVim.connect.Connect()
+    # host = pyVim.host.GetHostSystem(si)
+    # fss = host.configManager.storageSystem.fileSystemVolumeInfo.mountInfo
+    # vmfs_volume_info = [f.volume for f in fss if f.volume.name == datastore_name abd f.volume.type == "VMFS"]
+    # return vmfs_volume_info and vmfs_volume_info.local
+    return False
+
+
+def service_restart():
+    """Brute force restart the service, it does need it."""
+    # Note1: iNotify in the service would be better bu iNotify is only supported
+    # in ESX user space in 6.5+ (TBD: verify & file an issue).
+    print("Restarting the vmdkops service to pick up new configuration")
+    os.system("/etc/init.d/vmdk-opsd restart")
```

Comment:
yes, they will fail. There is a potential of inconsistencies though there are I am not sure all operations are transactional. But it is not new - same happens on '/etc/init.d/vmdk-ops stop'